Cargo.toml

```diff
@@ -70,6 +70,9 @@ tracing-subscriber = { version = "0.3.14", default-features = false, features =
 wasm-bindgen = "=0.2.87"
 wasm-bindgen-futures = "0.4.33"
 zeroize = "1.6.0"
+# 0.3.37 is the latest version that works with wasm-bindgen 0.2.87.  When the
+# dependency on wasm-bindgen is relaxed, this can be relaxed as well
+wasm-bindgen-test = "=0.3.37"
```

Comment:
Please rebase on top of `main`, it uses `wasm-bindgen` 0.2.89 now.

Reply:
The [`Cargo.toml`](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/blob/main/Cargo.toml#L70) still says 0.2.87

Reply:
that's because https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/82 still hasn't landed yet.

Reply:
Merged `main` and bumped the `wasm-bindgen-test` version.